src/System.Private.ServiceModel/tools/setupfiles/sync-pr.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+show_banner()
+{
+    echo ""
+    echo "WCF Remote Github Repo sync script"
+}
+
+show_usage() 
+{
+    echo "    A URL must be specified for the pull request synchronization URL"
+    echo "    Usage: $0 [sync-url]"
+    echo "    sync-url - URL on remote server for PR synchronization"
+    echo ""
+    echo "    Example:  $0 http://wcfci-sync-server/repo/sync.ashx"
+}
+
+run_request() 
+{
+    __sync_url=$1
+    __ghprbPullId=$2
+
```

Comment:
See comments about the CMD version.  Here the PR is an input parameter, which is what I would prefer for both.  It also belongs in the show_usage() description.